data-prepper-plugins/newline-codecs/src/main/java/org/opensearch/dataprepper/plugins/newlineinputcodec/NewlineDelimitedInputCodec.java

```diff
@@ -3,10 +3,11 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-package org.opensearch.dataprepper.plugins.source.codec;
+package org.opensearch.dataprepper.plugins.newlineinputcodec;
```

Comment:
I have the same comment on the package name. Rather than end with `newlineinputcodec`, end with either: `codec.newline` or simply `newline`.